client/packages/sourcegraph-extension-api/src/sourcegraph.d.ts

```diff
@@ -930,6 +930,134 @@ declare module 'sourcegraph' {
         provideView(context: DirectoryViewContext): ProviderResult<View>
     }
 
+    export interface ThemableFileDecorationStyle {
+        /** The CSS color property value for the text contet */
+        color?: string
+
+        /** Overwrite style for when the file is selected */
+        selectedColor?: string
+    }
+
+    /** A decoration attachment adds content after a {@link FileDecoration}. */
+    export interface FileDecorationAttachmentRenderOptions extends ThemableFileDecorationStyle {
+        /** Text value to be displayed. This value should be very short to prevent truncation */
+        value: string
```

Comment:
This is called `contentText` for line decorations
```suggestion
        contentText: string
```